src/pages/errorpages/Forbidden.tsx

```diff
@@ -1,20 +1,71 @@
-import { Box, Link as MuiLink, Typography } from '@mui/material';
-import { useTranslation } from 'react-i18next';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+import LockOutlineIcon from '@mui/icons-material/LockOutline';
+import LoginIcon from '@mui/icons-material/Login';
+import { Box, Button, SxProps, Typography } from '@mui/material';
+import { Trans, useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
 import { Link } from 'react-router';
+import { RootState } from '../../redux/store';
+import { LocalStorageKey } from '../../utils/constants';
+import { dataTestId } from '../../utils/dataTestIds';
+import { useAuthentication } from '../../utils/hooks/useAuthentication';
 import { UrlPathTemplate } from '../../utils/urlPaths';
 
+const buttonProps: SxProps = { textTransform: 'none', mt: '1rem', width: 'fit-content' };
+
 export const Forbidden = () => {
   const { t } = useTranslation();
+  const { handleLogin } = useAuthentication();
+  const user = useSelector((store: RootState) => store.user);
 
   return (
-    <Box data-testid="forbidden" sx={{ mt: '4rem' }}>
-      <Typography variant="h2" component="h1" sx={{ mb: '1rem' }}>
-        {t('authorization.forbidden')}
-      </Typography>
-      <Typography sx={{ mb: '1rem' }}>{t('authorization.forbidden_description')}</Typography>
-      <MuiLink component={Link} to={UrlPathTemplate.Root}>
-        <Typography>{t('authorization.back_to_home')}</Typography>
-      </MuiLink>
+    <Box data-testid="forbidden" sx={{ my: '4rem', display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
+      <div>
```

Comment:
Når man har en flex som har bare ett element finnes det ofte en enklere løsning. Virker som det her bare er gjort for å sentrere innholdet, i så fall kan man bruke `mx: 'auto'` på parent for det samme
```suggestion
    <Box data-testid="forbidden" sx={{ my: '4rem', mx: 'auto' }}>
```